docs/UserGuide.md

```diff
@@ -138,6 +160,18 @@ Examples:
 * `add n/Hershey s/Rufous Hummingbird i/193`
 * `add n/Lonesome George s/Galapagos Tortoise i/117 m/Flu f/1200`
 
+Example Usage:
+```
+add n/Kai Kai s/Giant Panda i/200 m/Sunstroke f/1230 f/1400
+```
+
+Expected Outcome:
+```
+New animal added: Kai Kai ID: 200 Species: Giant Panda Medical conditions: [Sunstroke] Feeding times: [1230][1400]
```

Comment:
Think the feeding times need to be changed to feed time after we edit it in the code as well.

Reply:
Yep sorry I'll wait for the code to be fixed first then I'll run all the commands again and paste the updated outcomes